packages/ast-spec/src/declaration/ExportNamedDeclaration/spec.ts

```diff
@@ -7,6 +7,7 @@ import type { ExportKind } from '../ExportAndImportKind';
 
 export interface ExportNamedDeclaration extends BaseNode {
   type: AST_NODE_TYPES.ExportNamedDeclaration;
+  // TODO(error handling) - ClassExpression is not valid here
```

Comment:
Are there tracking issue for these todos?

Reply:
#1852 is the issue for all  the "we should add AST errors" todos
I can update these comments with the issue number for reference